### PR TITLE
Add default infowindow template instead of trying to load "infowindow_light"

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Fixes CartoDB/cartodb#7421.

@xavijam 👍  or 👎 ?

### CODE REVIEW

File: src/geo/ui/infowindow-model.js [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
 
-/**
- * Usage:
- * var infowindowModel = new InfowindowModel({
- *   template_name: 'infowindow_light',
- *   latlng: [72, -45],
- *   offset: [100, 10]
- * });
- */
+var defaultInfowindowTemplate = [
```
Comment (xavijam) [inline src/geo/ui/infowindow-model.js]: Could we move to a template file?

**Comment by xavijam** [comment]
It sounds like a workaround, and I'm scared that there is no better solution and the same template is in several places :S.

File: src/geo/ui/infowindow-model.js (hunk 1 above)
Comment (alonsogarciapablo) [inline src/geo/ui/infowindow-model.js]: Moved in [4138bdca9d15ad3d3f9f6085db6a97f2c5f12d86](https://github.com/CartoDB/cartodb.js/pull/1270/commits/4138bdca9d15ad3d3f9f6085db6a97f2c5f12d86).

**Comment by xavijam** [comment]
yay! 👍

**Comment by alonsogarciapablo** [comment]
retest this please

**Pull request merged**